tests/unit/test_bruteforce.cpp

```diff
@@ -533,7 +504,8 @@ TEST_F(BruteForceTest, brute_force_reindexing_same_vector_different_id) {
 }
 
 TEST_F(BruteForceTest, test_delete_swap_block) {
-    size_t n = 6;
+    size_t n = 6;                 // idToLabelMapping initial size.
+    size_t initiail_capacity = 5; // idToLabelMapping initial size.
```

Comment:
5 or 6?

Reply:
moved n above the loop and removed the comment